src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/create_index_pattern_wizard.html

```diff
@@ -6,105 +6,92 @@
       class="kuiViewContent"
     >
       <!-- Intro -->
-      <div class="kuiVerticalRhythm">
-        <div class="kuiBar">
-          <h1 class="kuiTitle">
-            Create index pattern
-          </h1>
+      <h1 class="euiTitle euiTitle--medium">
+        Create index pattern
+      </h1>
 
-          <div class="kuiBarSection">
-            <!-- Include system indices -->
-            <label
-              ng-if="controller.isSystemIndicesCheckBoxVisible()"
-              class="kuiCheckBoxLabel"
-              for="indexPatternCreationIncludeSystemIndices"
-            >
-              <input
-                type="checkbox"
-                id="indexPatternCreationIncludeSystemIndices"
-                class="kuiCheckBox"
-                ng-model="controller.doesIncludeSystemIndices"
-                ng-change="controller.onIncludeSystemIndicesChange()"
-              >
+      <div class="euiSpacer euiSpacer--small"></div>
 
-              <span class="kuiCheckBoxLabel__text">
-                Include system indices
-              </span>
-            </label>
+      <div class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--justifyContentSpaceBetween euiFlexGroup--alignItemsEnd euiFlexGroup--responsive">
+        <div class="euiFlexItem euiFlexItem--flexGrowZero">
+          <div class="euiText">
+            <p class="euiTextColor euiTextColor--subdued">
+              Kibana uses index patterns to retrieve data from Elasticsearch indices for things like visualizations.
+            </p>
           </div>
         </div>
 
-        <p class="kuiText kuiSubduedText">
-          Kibana uses index patterns to retrieve data from Elasticsearch indices for things like visualizations.
-        </p>
+        <div class="euiFlexItem euiFlexItem--flexGrowZero">
+          <label
```

Comment:
We can convert this to an EUI checkbox by replacing this label and its contents with this:

```html
          <div
            class="euiCheckbox"
            ng-if="controller.isSystemIndicesCheckBoxVisible()"
          >
            <input
              type="checkbox"
              class="euiCheckbox__input"
              id="indexPatternCreationIncludeSystemIndices"
              ng-model="controller.doesIncludeSystemIndices"
              ng-change="controller.onIncludeSystemIndicesChange()"
            >
            <div class="euiCheckbox__square">
              <div class="euiCheckbox__check"></div>
            </div>
            <label
              class="euiCheckbox__label"
              for="indexPatternCreationIncludeSystemIndices"
            >
              Include system indices
            </label>
          </div>
```